rpm/htcondor-ce.spec

```diff
@@ -290,6 +295,8 @@ install -m 0755 -d -p $RPM_BUILD_ROOT/%{_sysconfdir}/condor-ce/bosco_override
 %attr(-,condor,condor) %dir %{_localstatedir}/lib/condor-ce/execute
 %attr(-,condor,condor) %dir %{_localstatedir}/lock/condor-ce
 %attr(1777,condor,condor) %dir %{_localstatedir}/lock/condor-ce/user
+%attr(0700,root,root) %dir %{_sysconfdir}/condor-ce/passwords.d
+%attr(0700,condor,condor) %dir %{_sysconfdir}/condor-ce/tokens.d
```

Comment:
Same as above: if both the central collectors and CEs need these dirs, this should go under `%files client`

Reply:
Done.